common/Printer.java

```diff
@@ -67,8 +68,11 @@ public void conceptRow(ConceptRow conceptRow, Transaction tx, boolean first) {
         out.println(conceptRowDisplayString(conceptRow, columnNames, columnsWidth, tx));
     }
 
-    public void json(JSON json) {
-        out.println(json.toString());
+    public void conceptDocument(JSON conceptDocument, boolean first) {
+        if (first) {
+            out.println(conceptDocumentDisplayHeader(QueryType.READ)); // TODO: Get the query type from the document when available
```

Comment:
Write queries are not supported by fetch, so it doesn't matter now. The query_type is not exposed to the Java driver at the moment. Might fix it in time before the release, might leave a small debt!